powerhome/playbook
# Enables Global Prop Classnames in React & Enables Dark Class Prop

#### Breaking Changes
None

#### Runway Ticket URL

https://nitro.powerhrg.com/runway/backlog_items/NUX-1185

#### How to test this

Add a `dark` prop to a react kit and see if the `dark` class appears at the end of the classname string in the DOM.

#### Checklist:
